Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_PlayerPhysics.cs

```diff
@@ -0,0 +1,115 @@
+namespace VRTK
```

Comment:
PlayerPhysics functionality has now all been merged into PlayerPresence. Not sure how to delete the VRTK_PlayerPhysics.cs file, but it's no longer needed.